tests/test_surface_tally/test_surface_tally.py

```diff
@@ -0,0 +1,164 @@
+                    cell_to_filter = openmc.CellFilter(cell2)
+
+                    cell_to_cell_tallies.append(openmc.Tally(tally_id=tally_index, name=str(cell1)+'-'+str(cell2)))
+                    cell_to_cell_tallies[tally_index].filters   = [cell_from_filter, cell_to_filter, energy_filter, polar_filter, azimuthal_filter]
+                    cell_to_cell_tallies[tally_index].scores    = ['current']
+                    tallies_file.append(cell_to_cell_tallies[tally_index])
+                    tally_index += 1
+
+                    # Cell from + surface filters for partial current
+                    surface_filter = openmc.SurfaceFilter([1])
+                    cell_to_cell_tallies.append(openmc.Tally(tally_id=tally_index, name=str(cell1)+'-surface1'))
+                    cell_to_cell_tallies[tally_index].filters   = [cell_from_filter, surface_filter, energy_filter, polar_filter, azimuthal_filter]
+                    cell_to_cell_tallies[tally_index].scores    = ['current']
+                    tallies_file.append(cell_to_cell_tallies[tally_index])
+                    tally_index += 1
+
+        # Surface filter on inner surface, for net current
+        surface_filter = openmc.SurfaceFilter([1])
+        surf_tally1 = openmc.Tally(tally_id=tally_index, name='net_cylinder')
+        surf_tally1.filters   = [surface_filter, energy_filter, polar_filter, azimuthal_filter]
+        surf_tally1.scores    = ['current']
+        tallies_file.append(surf_tally1)
+        tally_index += 1
+
+        # Surface filter on left surface, vacuum BC, for net current = leakage
+        surface_filter = openmc.SurfaceFilter([2])
+        surf_tally2 = openmc.Tally(tally_id=tally_index, name='leakage_left')
+        surf_tally2.filters   = [surface_filter, energy_filter, polar_filter, azimuthal_filter]
+        surf_tally2.scores    = ['current']
+        tallies_file.append(surf_tally2)
+        tally_index += 1
+
+        # Surface filter on right surface, reflective, current tally doesnt pick up the zero net current though
+        surface_filter = openmc.SurfaceFilter([3])
+        surf_tally3 = openmc.Tally(tally_id=tally_index, name='net_right')
+        surf_tally3.filters   = [surface_filter, energy_filter]
+        surf_tally3.scores    = ['current']
+        tallies_file.append(surf_tally3)
+        tally_index += 1
+
+        tallies_file.export_to_xml()
+
+    def _get_results(self):
+        """Digest info in the statepoint and return as a string."""
+        # Read the statepoint file.
+        sp = openmc.StatePoint(self._sp_name)
+
+        # Extract the tally data as a Pandas DataFrame.
+        df = pd.DataFrame()
+        for t in sp.tallies.values():
+            df = df.append(t.get_pandas_dataframe(), ignore_index=True)
+
+        # Extract the relevant data as a CSV string.
+        cols = ('d_material', 'd_nuclide', 'd_variable', 'score', 'mean',
+                'std. dev.')
```

Comment:
I would just make this `cols = ('mean', 'std. dev.')`. The first three variables don't apply to this test and 'score' is always 'current', so not very useful.